libs/remotion/templates/src/lib/foregrounds/Emgg.template.tsx

```diff
@@ -2,12 +2,16 @@ import { ForegroundComponents } from '@pulsebanner/remotion/components';
 import { ForegroundForms } from '@pulsebanner/remotion/forms';
 import { Layer } from '../Layer';
 
-export const TwitchStream: Layer<typeof ForegroundComponents.TwitchStream> = {
```

Comment:
what were we using this file for before?

Reply:
nothing